### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by baskettcase** [Droeftoeter/pokapi#13]
Just a quick question.. is the package finished enough to be able to get a list of the pokemon in inventory and their stats?

**Comment by xmillies** [comment]
Could you add header (use xxx\xxx; and fake values) in your sample ?
What about deviceId ?

**Comment by Droeftoeter** [comment]
@baskettcase You should be able to retrieve those, note that with Pokapi you get the raw response message for now. Ie the `GetInventoryResponse`-object.

**Comment by Droeftoeter** [comment]
@xmillies I will add them in the next commit, I've included them in this comment:

`use Pokapi\API;
use Pokapi\Version\Latest;
use Pokapi\Hashing\Pogodev;
use Pokapi\Authentication\TrainersClub;
use Pokapi\Request\DeviceInfo;
use Pokapi\Request\Position;`

For the DeviceId, you can generate a default set with a custom identifier by using `Pokapi\Request\DeviceInfo::getDefault($yourUniqueIdentifier);`. The default is an Android phone.

**Comment by baskettcase** [comment]
Ok thanks @Droeftoeter Ill mess around with it and see what I can do. Im on iOS so Ill look through it and see what kind of structure I might need to get as close as possible to my current device. And Google sign in.

**Comment by baskettcase** [comment]
Is there any way to reduce the PHP version requirements?

**Comment by Ni42** [comment]
You should just upgrade to 7.1, what's stopping you?

**Comment by baskettcase** [comment]
because I manage way too many projects for people and I need to mirror as close as possible most people's hosting environments and I don't think a single one even uses 7.0 let alone 7.1 at this point. It's ok, I don't need to use this package, I just figured if it was possible to reduce the PHP version then more people could use it. :)

**Comment by jaspervdm** [comment]
Its entirely possible to run multiple PHP versions next to each other though

**Comment by baskettcase** [comment]
true true

**Comment by xmillies** [comment]
You can use docker to isolate your php (or php-fpm) version.
[https://hub.docker.com/_/php/](url)

**Comment by baskettcase** [comment]
heh thanks :) So im guessing from all the discussion, the answer is no? :)

**Comment by Droeftoeter** [comment]
Yeah, kinda. PHP 7.1 was the latest version when I restarted on this project. I have no plans to downgrade at this time.

**Comment by baskettcase** [comment]
Ok thank you. :)

**Comment by xmillies** [comment]
@Droeftoeter Do you that you could make an PR on NicklasWallgren/PokemonGoAPI-PHP source to work (with or withou hashing server) ?

**Comment by xmillies** [comment]
Could you add getPokemons() ?
